helpers/handlebars/middleware.js

```diff
@@ -112,6 +112,16 @@ module.exports = (req, res, next) => {
         introText: "Hier gelangst du zur Materialsuche, bei der du in der Datenbank der Schul-Cloud nach Materialien für deine Unterrichtsstunde suchen kannst."
     }];
 
+    // Extensions Feature Toggle
+    const extensionsEnabled = process.env.SC_THEME === "n21";
```

Comment:
Bitte einen eigenen Feature-Toggle, sonst müssen wir für jede Instanz, die auch Add-ons möchte den Code ändern.

Reply:
Did that.